src/app/reporting/Engine.cpp

```diff
@@ -37,6 +37,7 @@
 #include <lib/support/CodeUtils.h>
 #include <optional>
 #include <protocols/interaction_model/StatusCode.h>
+#include <regex.h>
```

Comment:
Why is regex.h needed?  That seems very odd.

Reply:
auto-include (probably by a typo) ... will fix